pkg/driver/device_manager.go

```diff
@@ -69,6 +71,8 @@ type DeviceManager interface {
 	WriteInfo(*info.ZInfoMsg) error
 	// WriteLogs write a LogBundle message
 	WriteLogs(*logs.LogBundle) error
+	// WriteAppInstanceLogs write a AppInstanceLogBundle message for instance UUID
+	WriteAppInstanceLogs(uuid.UUID, uuid.UUID, *logs.AppInstanceLogBundle) error
```

Comment:
* do we need to pass the instance UUID as well? Is that, at least, not available from the `AppInstanceLogBundle`?
* can you add to the comment why we need to pass the app UUID (as the app log message does not indicate it)?
* if we need to pass the instance UUID, please put named parameters in the `interface`, so consumers can know which one is first

Reply:
No uuid inside https://github.com/lf-edge/eve/blob/0811608f0a67121028c55c76f6c767d5311b32a6/api/go/logs/log.pb.go#L228

Reply:
OK, in that case, let's make the interface be explicit about the named args, so we know which is which, and make the comment more explicit.